### PR TITLE
Issues resolved for company profile and GSTIN and manufacturing

### PR DESCRIPTION
**Pull request by rohit-walkover**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/manufacturing/edit/mf.edit.component.ts [hunk 1]
```diff
@@ -85,12 +85,12 @@ export class MfEditComponent implements OnInit {
...
         this.isStockWithRateInprogress$ = this.store.pipe(select(state => state.manufacturing.isStockWithRateInprogress), takeUntil(this.destroyed$));
 
-        this.groupsList$ = this.store.select(p => p.general.groupswithaccounts).pipe(takeUntil(this.destroyed$));
+        this.groupsList$ = this.store.pipe(select(p => p.general.groupswithaccounts), takeUntil(this.destroyed$));
 
         this.manufacturingDetails = new ManufacturingItemRequest();
         this.initializeOtherExpenseObj();
         // Update/Delete condition
-        this.store.select(p => p.manufacturing).pipe(takeUntil(this.destroyed$)).subscribe((o: any) => {
+        this.store.pipe(select(p => p.manufacturing), takeUntil(this.destroyed$)).subscribe((o: any) => {
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/manufacturing/edit/mf.edit.component.ts:93]: rename variable p and o

File: apps/web-giddh/src/app/settings/profile/setting.profile.component.ts [hunk 2]
```diff
@@ -99,6 +102,8 @@ export class SettingProfileComponent implements OnInit, OnDestroy {
         digitAfterDecimal.map(d => {
             this.decimalDigitSource.push({ value: d.value, label: d.name });
         });
+        this.getCompanyProfileInProgress$ = this.store.pipe(select(s => s.settings.getProfileInProgress),takeUntil(this.destroyed$));
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/settings/profile/setting.profile.component.ts:105]: rename variable s

File: apps/web-giddh/src/app/store/Settings/Settings.reducer.ts [hunk 3]
```diff
@@ -129,7 +130,8 @@ export const initialState: SettingsState = {
     isPaymentAdditionSuccess: false,
     isPaymentUpdationSuccess: false,
     taxes: null,
-    branchRemoved: false
+    branchRemoved: false,
+    getProfileInProgress: false
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/store/Settings/Settings.reducer.ts:134]: JS DOC

File: apps/web-giddh/src/app/settings/profile/setting.profile.component.ts (hunk 2 above)
Comment (prateek-giddh) [inline apps/web-giddh/src/app/settings/profile/setting.profile.component.ts:105]: Avoid single character name in variables

**Pull request merged**